### PR TITLE
[feat]: [CDS-85932] : update Service/Env/Infra static v1 yaml schema

### PR DESCRIPTION
**Pull request by tchaurasiya**
Changes
Changing title of svc + env + infra
deleting infraDefintion (without 's' from env schema)
changing name field to ref in svc + env
changing identifier to id in infra

### CODE REVIEW

File: v1/pipeline/stages/cd/infra-structure-definition-yaml.yaml [hunk 1]
```diff
@@ -15,5 +15,5 @@ properties:
   metadata:
     type: string
   description:
```
Comment (markoharness) [inline v1/pipeline/stages/cd/infra-structure-definition-yaml.yaml:17]: same as previous

File: v1/pipeline/stages/cd/infra-structure-definition-yaml.yaml [hunk 2]
```diff
@@ -15,5 +15,5 @@ properties:
...
-    desc: This is the description for InfraStructureDefinitionYaml
+    desc: This is the description for SimplifiedInfraDefYaml
```
Comment (markoharness) [inline v1/pipeline/stages/cd/infra-structure-definition-yaml.yaml:18]: same as previous

File: v1/pipeline/stages/cd/service-yaml-v2.yaml [hunk 3]
```diff
@@ -1,16 +1,16 @@
...
       pattern: (^<\+input>((\.)((executionInput\(\))|(allowedValues|default|regex)\(.+?\)))*$|^$)
-  name:
+  ref:
     type: string
     pattern: ^(?=\s*\S).*$
   use-from-stage:
     $ref: service-use-from-stage-v2.yaml
   description:
-    desc: This is the description for ServiceYamlV2
+    desc: This is the description for SimplifiedServiceYaml
```
Comment (markoharness) [inline v1/pipeline/stages/cd/service-yaml-v2.yaml:15]: same as previous

File: v1/pipeline/stages/cd/service-yaml-v2.yaml [hunk 4]
```diff
@@ -1,16 +1,16 @@
-title: ServiceYamlV2
+title: SimplifiedServiceYamlV2
```
Comment (markoharness) [inline v1/pipeline/stages/cd/service-yaml-v2.yaml:1]: Should it be SimplifiedServiceYamlV2 or SimplifiedServiceYaml?

**Review by markoharness — DISMISSED** [review]

**Review by archit-harness — APPROVED** [review]

**Review by markoharness — APPROVED** [review]

**Review by yogesh-chauhan — APPROVED** [review]

**Pull request merged**